Makefile

```diff
@@ -7,7 +7,6 @@ SHORT_NAME ?= etcd
 export GOARCH ?= amd64
 export GOOS ?= linux
 export MANIFESTS ?= ./manifests
-export DEV_REGISTRY ?= "$(shell docker-machine ip deis 2>&1):5000"
 export DEIS_REGISTRY ?= ${DEV_REGISTRY}/
```

Comment:
for my reference, do people generally have preference for using `export` in makefiles for vars? I'm leveling up my makefile-fu...